bazel/openjph_module_dot_bazel.patch

```diff
@@ -3,7 +3,7 @@
 @@ -0,0 +1,8 @@
 +module(
 +    name = "openjph",
-+    version = "0.21.0",
++    version = "0.21.2",
 +    compatibility_level = 1,
 +)
 +
```

Comment:
change bazel_dep(name = "rules_cc", version = "0.1.0") to bazel_dep(name = "rules_cc", version = "0.1.1")